web/src/main/java/com/microsoft/applicationinsights/web/internal/correlation/tracecontext/Tracestate.java

```diff
@@ -11,27 +15,97 @@
  *
  * Implementations can add vendor specific details here.
  */
-@Experimental
 public class Tracestate {
 
     /**
-     * Field to capture tracestate header
+     * Internal representation of the tracestate
+     */
+    private LinkedHashMap<String, String> internalList = new LinkedHashMap<String, String>(32);
```

Comment:
A good practice to follow is use Interface. 
`Map<String, String> internalList = new LinkedHashMap<>(32)`. You also need not specify the types in diamond operator twice.  By the way, LinkedHashMap is used to maintain insertion order. Do you need to maintain that order here?

Reply:
Is 32 the max limit according to spec?

Reply:
Yes, https://github.com/w3c/trace-context/blob/master/trace_context/HTTP_HEADER_FORMAT.md#header-value
```
There can be a maximum of 32 list-members in a list.
```